README.md

```diff
@@ -16,12 +20,116 @@ For npm users:
 npm install @istreamplanet/pebble-formik formik @istreamplanet/pebble node-sass react-router-dom --save
 ```
 
-Include the Pebble foundation stylesheet in your layout template.
+Include the Pebble foundation stylesheet in your layout template. (This is usually added at the outer most content wrapper of an app)
 
 ```shell
 import '@istreamplanet/pebble/dist/Styles/foundation.scss';
 ```
 
+Import the components into your application
+
+```shell
+import { FieldSelect, FormCard, FormLayoutRow } from '@istreamplanet/pebble-formik'`
+```
+
+Then add them in your React component as you would any other:
+
+**The [FormCard](https://github.com/istreamlabs/pebble-formik/blob/main/src/FormCard.tsx) component is a great all-in-one form wrapper component. Please try it before building a Formik form from scratch.**
+
+```shell
+const MyComponent = () => (
+  <Block>
+    <FormCard initialValues={initialValues} onSubmit={handleSubmit} validate={validate}>
+      <FormLayoutRow>
+        <FieldSelect id="foo" name="foo" label="Foo" options={FooOptions} />
+      </FormLayoutRow>
+    </FormCard>
+  </Block>
+)
+```
+
+## Components
+
+### [FormCard](https://github.com/istreamlabs/pebble-formik/blob/main/src/FormCard.tsx)
+
+FormCard was designed to be an all-in-one form rapper, including a prompt message if the user tries to navigate away from the form and a submit and reset button in the footer. (see component for all of the props)
```

Comment:
Looks like I missed this the first pass, should be wrapper, not rapper.

Reply:
hahaha!!! oops